ssl/record/methods/tls1_meth.c

```diff
@@ -162,6 +162,7 @@ static int tls1_cipher(OSSL_RECORD_LAYER *rl, SSL3_RECORD *recs, size_t n_recs,
     EVP_CIPHER_CTX *ds;
     size_t reclen[SSL_MAX_PIPELINES];
     unsigned char buf[SSL_MAX_PIPELINES][EVP_AEAD_TLS1_AAD_LEN];
+    unsigned char *data[SSL_MAX_PIPELINES];
```

Comment:
Why was this moved to the outer scope? It's the only change in this function.

Reply:
This was due to use-after-scope error picked up by asan. The buffers set by `EVP_CTRL_SET_PIPELINE_OUTPUT_BUFS` must still be valid at the point that we call `EVP_CipherUpdate` - which was not the case when this was in its original location.